app/config.js

```diff
@@ -34,7 +34,8 @@
     loggingURL: "",
     metricsURL: ""
   };
-
+  // Additional support only for Chinese now: zh-CN
```

Comment:
I wouldn't use the word support here since this is still going to be a work in progress, maybe say something like:
Internationalization is currently under development, the default supported language is English